tidb-cloud/set-up-private-endpoint-connections.md

```diff
@@ -34,137 +34,141 @@ In most scenarios, you are recommended to use private endpoint connection over V
-    <div label="Use AWS Console">
+Then create an AWS interface endpoint either in the AWS Management Console or using the AWS CLI.
 
-    To use the AWS Management Console to create the VPC interface endpoint, perform the following steps:
+<SimpleTab>
+<div label="Use AWS Console">
 
-    1. In your AWS Management Console, go to **VPC** > **Endpoints**, and click **Create Endpoint** in the upper-right corner. The **Create endpoint** page is displayed.
+To use the AWS Management Console to create the VPC interface endpoint, perform the following steps:
 
-        ![Create endpoint](/media/tidb-cloud/private-endpoint/create-endpoint-1.png)
+1. Go to **VPC** > **Endpoints**.
+2. Click **Create Endpoint**.
 
-    2. Under **Service category**, select **Other endpoint services**.
-    3. Under **Service settings**, enter the endpoint service name you have obtained from the **Interface endpoint** page of the TiDB Cloud console, and click **Verify service**.
+    The **Create endpoint** page is displayed.
 
-        ![Verify endpoint service](/media/tidb-cloud/private-endpoint/create-endpoint-2.png)
+    <img src="../media/tidb-cloud/private-endpoint/create-endpoint-2.png" width="70%" auto="Verify endpoint service" />
 
-    4. After the service name is verified, under **VPC**, select your VPC in the drop-down list. Then the pre-populated **Subnets** area is displayed.
-    5. In the **Subnets** area, select the availability zones where your TiDB cluster is located. Then click **Create endpoint** at the bottom of the page.
-
-        ![Create endpoint service 2](/media/tidb-cloud/private-endpoint/create-endpoint-3.png)
+3. Select **Other endpoint services**.
+4. Enter the endpoint service name.
+5. Click **Verify service**.
+6. Select your VPC in the drop-down list.
+7. Select the availability zones where your TiDB cluster is located in the **Subnets** area.
 
     > **Tip:**
     >
     > If your service is spanning across more than three availability zones (AZs), you might not be able to select AZs in the **Subnets** area. This issue occurs when there is an extra AZ in your selected region in addition to the AZs where your TiDB cluster is located. In this case, contact [PingCAP Technical Support](https://docs.pingcap.com/tidbcloud/tidb-cloud-support).
 
-    </div>
-    <div label="AWS CLI">
+8. Select your security group properly in the **Security groups** area.
+9. Click **Create endpoint**.
 
-    To use the AWS CLI to create a VPC interface endpoint, perform the following steps:
+</div>
+<div label="Use AWS CLI">
 
-    1. Install AWS Command Line Interface (AWS CLI).
+To use the AWS CLI to create a VPC interface endpoint, perform the following steps:
 
-        ```bash
-        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
-        unzip awscliv2.zip
-        sudo ./aws/install
-        ```
+1. Fill in the **VPC ID** and **Subnet IDs** fields. You can get the IDs from your AWS Management Console.
```

Comment:
which page are the **VPC ID** and **Subnet IDs** fields located?

Reply:
```suggestion
1. Fill in the **VPC ID** and **Subnet IDs** fields on the private endpoint creation page. You can get the IDs from your AWS Management Console.
```